src/components/K8s.vue

```diff
@@ -51,8 +63,56 @@ export default {
   computed: {
     cannotReset: function() {
       return (this.state !== K8s.State.STARTED && this.state !== K8s.State.READY);
+    },
+
+    memoryValueIsValid: function() {
+      let value = this.settings.kubernetes.memoryInGB;
+      let numericValue;
+      if (typeof(value) !== "number") {
+        numericValue = parseFloat(value, 10);
+        if (isNaN(numericValue)) {
+          return false;
+        }
+      }
+      if (numericValue < 2 ||
+          (numericValue > this.availMemoryInGB && this.availMemoryInGB)) {
```

Comment:
Will the UI do the correct thing if we end up on a machine with 2GB total memory? (Actually, I'm not even sure what the correct thing would be on such a system…)

Reply:
The UI (when sliderized) will lock the widget. You'll probably have other problems trying to run rd on such a machine.